src/ResponseBuilder.php

```diff
@@ -20,7 +20,13 @@ private function getNormalizedResponse(string $body)
         $decodedResponse = $this->getJsonDecodedResponse($body);
         
         if (!array_key_exists('data', $decodedResponse)) {
-            throw new \UnexpectedValueException('Invalid GraphQL JSON response.');
+            $message = 'Invalid GraphQL JSON response.';
```

Comment:
In this case it would not be an invalid JSON response: the response would be valid, but an error or more occurred. What do you think about moving out the condition checking for the key errors, at the same level of the one checking for "data"? Then we could output the whole JSON below that key (without checking for the first message), and in case neither key "data" nor "errors" exist, we could throw "Invalid GraphQL JSON response".